static/js/components/PrivacyForm.js

```diff
@@ -0,0 +1,26 @@
+/* global SETTINGS: false */
+import React from 'react';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+
+import ProfileFormFields from '../util/ProfileFormFields';
+
+class PrivacyForm extends ProfileFormFields {
+  static propTypes = {
+    profile:        React.PropTypes.object,
+    ui:             React.PropTypes.object,
+    updateProfile:  React.PropTypes.func,
+  };
+
+  render() {
+    return (
+      <Grid className="profile-tab-grid privacy-form">
```

Comment:
I think it would overall make more sense to have the parent component worry about layout, and have the `PrivacyForm` just return basically the form field and wire everything up correctly, so when I want to plop it on, say, `PrivacyTab` I'd just do somehting like

```js
<Cell col={12}>
  <PrivacyForm {...this.props} otherProps={...} />
</Cell>
```
That way the layout is the responsibility of the context where we actually want to render it, and the `PrivacyForm` component ends up being a bit more flexible and minimal.